patches/0002-remove_min_reviwers_defaults.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/azuredevops/internal/service/policy/branch/resource_branchpolicy_min_reviewers.go b/azuredevops/internal/service/policy/branch/resource_branchpolicy_min_reviewers.go
+index f51caa42..af9b2ae9 100644
+--- a/azuredevops/internal/service/policy/branch/resource_branchpolicy_min_reviewers.go
++++ b/azuredevops/internal/service/policy/branch/resource_branchpolicy_min_reviewers.go
+@@ -38,7 +38,6 @@ func ResourceBranchPolicyMinReviewers() *schema.Resource {
+ 	settingsSchema["on_last_iteration_require_vote"] = &schema.Schema{
+ 		Type:          schema.TypeBool,
+ 		Optional:      true,
+-		Default:       false,
```

Comment:
This might be a no-op I think. Go will assume Default==false if you don't specify it, so this change does not change it, but no harm done.

Reply:
`Default` is typed as `interface{}` and they seem to do some checking on the type here - without this patch the test fails, so it is certainly not a noop.